selfdrive/sensord/sensors/bmx055_gyro.cc

```diff
@@ -26,6 +27,13 @@ int BMX055_Gyro::init() {
     goto fail;
   }
 
+  ret = set_register(BMX055_GYRO_I2C_REG_LPM1, BMX055_GYRO_NORMAL_MODE);
+  if (ret < 0) {
+    goto fail;
+  }
+  // bmx055 gyro has a 30ms wakeup time from deep suspend mode
+  util::sleep_for(50);
```

Comment:
can we not write to the registers during this time?

Reply:
nope, this is behavior is a bit weird, if we dont give the bmx time after setting this bit the change will not be applied and the bmx gyro stays in deep suspend mode

Reply:
is there a register we can read instead?

Reply:
unfortunatelly, I havent found anything about this behaviour in the data sheet, except the wakeup time...
we could busy wait if the values got applied to the register?

Reply:
this is fine, we're not gonna use these anyway in a bit